mos-platform/nes/rompoke/misc/generate-rompoke-mapper-linkscript.lua

```diff
@@ -0,0 +1,152 @@
+-- Copyright (c) 2023 Adrian "asie" Siekierka
+--
+-- Licensed under the Apache License, Version 2.0 with LLVM Exceptions,
+-- See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
+-- information.
+
+local CONFIG_MAP = {
```

Comment:
It looks like if we go this way, adding new mappers with bank conflicts would practically need to involve modifying this script. Accordingly, this is a major part of the porting interface for the NES, so it needs a lot of cleanup, additional documentation, or both, such that it's possible to relatively quickly get a sense of how to do so. At least on my end, I've only formed a very, vague picture of how this works after studying it for 5 minutes or so.

Reply:
Should be taken care of, now.